tests/js/realm-tests.js

```diff
@@ -1140,20 +1140,18 @@ module.exports = {
       let object = realm.create("CustomObject", { intCol: 1 });
       TestCase.assertTrue(object instanceof CustomObject);
       TestCase.assertTrue(Object.getPrototypeOf(object) == CustomObject.prototype);
-      TestCase.assertEqual(customCreated, 1);
+      TestCase.assertEqual(customCreated, 0);
 
       // Should be able to create object by passing in constructor.
       object = realm.create(CustomObject, { intCol: 2 });
       TestCase.assertTrue(object instanceof CustomObject);
       TestCase.assertTrue(Object.getPrototypeOf(object) == CustomObject.prototype);
-      TestCase.assertEqual(customCreated, 2);
+      TestCase.assertEqual(customCreated, 0);
     });
 
-    TestCase.assertThrowsContaining(() => {
-      realm.write(() => {
-        realm.create("InvalidObject", { intCol: 1 });
-      });
-    }, "Realm object constructor must not return another value");
+    realm.write(() => {
+      realm.create("InvalidObject", { intCol: 1 });
+    });
```

Comment:
This is no longer an issue since the `InvalidObject` function is no longer called on creation.